File: VB/WindowsApplication1/XtraForm1.vb
Imports DevExpress.Utils

Namespace WindowsApplication1

    Public Partial Class XtraForm1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Protected Overrides Function CreateFormBorderPainter() As DevExpress.Skins.XtraForm.FormPainter
            Dim formCaptionAlignment As HorzAlignment = HorzAlignment.Center
            Return New CustomFormPainter(Me, LookAndFeel, formCaptionAlignment)
        End Function
    End Class
End Namespace

File: CS/WindowsApplication1/XtraForm1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.Utils;

namespace WindowsApplication1 {
    public partial class XtraForm1 : DevExpress.XtraEditors.XtraForm {
        public XtraForm1() {
            InitializeComponent();
        }

        protected override DevExpress.Skins.XtraForm.FormPainter CreateFormBorderPainter() {
            HorzAlignment formCaptionAlignment = HorzAlignment.Center;
            return new CustomFormPainter(this, LookAndFeel, formCaptionAlignment);
        }
    }
}
